services/tftp/pom.xml

```diff
@@ -12,5 +12,47 @@
 
     <artifactId>manager-service-tftp</artifactId>
 
-    
+    <dependencies>
+
+        <!--
+        OUR COMMON API THAT ALL SERVICES SHARE
+        -->
+        <dependency>
+            <groupId>org.openthinclient</groupId>
+            <artifactId>manager-service-common</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
+        </dependency>
+
+        <!-- spring -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+        </dependency>
```

Comment:
Are all those spring dependencies actually required? The code of the module seems to be spring free. Therefore there should not be the need to include springdependencies in here.